node/settings.go

```diff
@@ -78,3 +79,11 @@ func WithHeaderConstructFn(construct header.ConstructFn) Option {
 		sets.opts = append(sets.opts, fx.Replace(construct))
 	}
 }
+
+// WithCustomKeyringSigner overrides the default keyring signer constructed
+// by the node.
+func WithCustomKeyringSigner(signer *apptypes.KeyringSigner) Option {
```

Comment:
nit: With...Option already assumes customizability, so `Custom` is redundant, or we should add Custom to each existing options for consistency

Reply:
kk will remove Custom, np